packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -409,6 +409,12 @@
       <artifactId>hive-service</artifactId>
       <version>${hive.version}</version>
       <scope>${utilities.bundle.hive.scope}</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.eclipse.jetty</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
You have already excluded jetty while defining the dependencies in root POM. You should not have to do that individually in each and every POM file now.

Reply:
Yeah, @rahil-c let's try to minimize the exclusions.

Reply:
Will try this out